src/Mod/Path/PathScripts/PathSurface.py

```diff
@@ -683,27 +660,8 @@ def beginLayerCommand(pnt, lc):
             if ignWF is False:
                 prcs = True
             else:  # ignWF is TRUE
-                if obj.LayerMode == 'Single-pass':
-                    if ignoreMap[i] > minIgnVal:
-                        if ignoreMap[i] == 1:
-                            pnt.z = obj.IgnoreWasteDepth
-                        if prcs is False:
-                            # Move cutter to current xy position
-                            output.append(Path.Command('G0', {'X': pnt.x, 'Y': pnt.y, 'F': self.horizRapid}))
-                        prcs = True
-                    else:
-                        if prcs is True:
-                            # Raise cutter to safe height
-                            output.append(Path.Command('G0', {'Z': obj.SafeHeight.Value, 'F': self.vertRapid}))
-                        prcs = False
-                else:  # Multi-pass mode
-                    if ignoreMap[i] > minIgnVal:
-                        if prcs is False:
-                            # Move cutter to current xy position
-                            output.append(Path.Command('G0', {'X': pnt.x, 'Y': pnt.y, 'F': self.horizRapid}))
-                        prcs = True
-                    else:
-                        prcs = False
+                prcs = True
+                pass
```

Comment:
`pass` is not needed here because the `else` clause is not empty. This will be reported by the LGTM code checker.

Reply:
Thanks for the review.  I corrected this in recent commit.  I appreciate the involvement, Sir.